src/ArcanaPrime.sol

```diff
@@ -361,6 +378,11 @@ contract ArcanaPrime is ERC721AQueryable, ERC721ABurnable, Ownable, OperatorFilt
         }
     }
 
+    // Future-proof
+    function setMintSupply(uint256 _mintSupply) public onlyOwner isBelowMaxSupply(_mintSupply) {
```

Comment:
```suggestion
    function setMintSupply(uint256 _mintSupply) external onlyOwner isBelowMaxSupply(_mintSupply) {
```

Reply:
Also this method can be used for "pausing" of the mints by just setting `mintSupply` to 0. It is not a pretty solution but it would save gas and gas is a priority for such an implementation

Reply:
got it!